### ISSUE TITLE
How-to

### ISSUE THREAD

**Issue by mopsusm** [jpterry/pcapsipdump#1]
I don't know that this is the best place for this, but didn't know where else to post. This looks like it will do exactly what I am looking for, the only problem is I can't figure out how to install it. Are there any detailed walk throughs anywhere? I've tried in both Debian6 and Fedora 20. When I've tried to make install-debian it gives an error. 

If this is the best place to get help I'll post back with more detail, or if there is somewhere else I should be going for help, please point me in the right direction.

**Comment by jnankin** [comment]
Yes it appears that instal-debian doesn't even exist.  Additionally you'd have to edit the Makefile to get this to compile:

 $(CC) $(CPPFLAGS) $(LDFLAGS) $(DEFS) pcapsipdump.cpp calltable.cpp -o pcapsipdump $(LIBS)

I had to move the LIBS to the end instead of at the beginning of the line
